doc/going-further/extending-castor/events.md

```diff
@@ -6,22 +6,50 @@ application lifecycle.
 
 ## Registering a listener
 
-You can register a listener inside your Castor project by using the
-`Castor\Attribute\AsListener` attribute. This attribute allows you to specify
-the targeted event and the priority of this listener.
+You can register a listener inside your Castor project by using the `Castor\Attribute\AsListener` attribute as a **function** (not a class method). This attribute allows you to specify the targeted event and the priority of this listener.
```

Comment:
I'm not sure it's necessary. We never use class nor method in castor. The example bellow show to do it. Did you really face this issue?

Reply:
So I saw the event system, ran bin/console make:listener, and tweaked the resulting file.  It took be a while to register that it was a function event, not a method event.

As it was missing the AsListener function 'use' statement (which I added in the PR), I thought perhaps it was a typo.  Anyway, I went down the wrong path because I hadn't registered that they were functions, not methods.  That's why I spelled it out.

Reply:
> ran bin/console make:listener,

The root of the issue was found here. I'm really sorry you misunderstood the difference between Castor and Symfony. 

I think this issue is really an edge case and shouldn't be covered in the documentation. It would just cause more confusion.

That's why I'd like to close this PR. Again, I'm sorry for the time you lost here. It's never fun. But thank you very much for taking the time to contribute to this project.